SolutionColor/TitleBarColorController.cs

```diff
@@ -42,6 +47,14 @@ static public TitleBarColorController CreateFromWindow(Window window)
                     // However docked-out code windows are just like in previous versions.
                     var dockPanel = VisualTreeHelper.GetChild(newController.titleBarContainer, 0);
                     newController.titleBarTextBox = VisualTreeHelper.GetChild(dockPanel, 3) as TextBlock;
+
+                    // In VS2019+ the main menu has been integrated with the title bar.
+                    // We can set the opacity to 0 and color the text as we did/do with the title text.
+                    newController.mainMenuControl = GetDecendantFirstInLine(newController.titleBarContainer, 6);
```

Comment:
This throws System.ArgumentOutOfRangeException (from `VisualTreeHelper.GetChild`) when I run it in VS2015, needs to have more safeguards.
Exception is caught and ignored, but means that the plugin is non-functional for vs2015